lib/opal/cli_runners/compiler.rb

```diff
@@ -91,7 +91,7 @@ def watch_compile
   end
 
   def reexec
-    system($0, *OriginalARGV) && exit
+    Process.kill('USR2', Process.pid)
```

Comment:
From what I distinctly remember, Windows may have issues with signals, eg. it doesn't support SIGTERM, I wonder if it does support USR2